build/release/build.sh

```diff
@@ -68,7 +68,7 @@ function build_all {
     PROFILES_ARR+=(search-all-tests)
     PROFILES_ARR+=(jdbc-all-tests)
     PROFILES=$(IFS=, ; echo "${PROFILES_ARR[*]}")
-    _mvn 'fhir-parent' "-Pdeploy-bintray,fhir-javadocs,fhir-validation-distribution,fhir-ig-carin-bb,fhir-ig-davinci-pdex-plan-net,fhir-ig-mcode,fhir-ig-us-core,fhir-term,${PROFILES}"
+    _mvn 'fhir-parent' "-Pdeploy-bintray,fhir-javadocs,fhir-validation-distribution,fhir-ig-carin-bb,fhir-ig-davinci-pdex-plan-net,fhir-ig-davinci-pdex-plan-net,fhir-ig-mcode,fhir-ig-us-core,fhir-term,${PROFILES}"
```

Comment:
is it bad that I don't even remember what these maven profiles do?  is it documented somewhere?

Reply:
Yes, documented in CreateValidationAssembly.md it dumps the profile in the validation zip.

Reply:
ah yes, now I see it in CreateFHIRValidationAssembly.md ...these were introduced to break a potential dependency cycle so that we can include the profiles in the "validation-assembly"

Reply:
right